### PR TITLE
Add article for free trials

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
New article with information on free trials and their limitations.

### CODE REVIEW

File: docs/platform/concepts/free-trial.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* Up to 10 VMs
+
+* 1 :doc:`Virtual Private Cloud (VPC) </docs/platform/howto/manage-vpc-peering>`
+
+* Up to 10 VPC peering connections
+
+
+There are some limitations:
+
+* Trial projects are automatically created and cannot be renamed
```
Comment (carabaestlein) [inline docs/platform/concepts/free-trial.rst:24]: Technically you can rename the project during signup, but at that point it isn't a trial project yet. Maybe we could rephrase this to: "You can only create a trial in your default project, i.e. the first project that is created when you sign up."

File: docs/platform/concepts/free-trial.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Trials include:
+
+* Up to 10 VMs
+
+* 1 :doc:`Virtual Private Cloud (VPC) </docs/platform/howto/manage-vpc-peering>`
+
+* Up to 10 VPC peering connections
+
+
+There are some limitations:
```
Comment (carabaestlein) [inline docs/platform/concepts/free-trial.rst:22]: Is it also worth adding explicitly that you can only have one trial?

File: docs/platform/concepts/free-trial.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+==============
+
+Aiven offers a free trial for 30 days to explore the platform and the `Aiven services <https://docs.aiven.io/>`_. You don't need a credit card to sign up.
+
+If you create services on the :doc:`free plan </docs/platform/concepts/free-plan>`, they will continue to run alongside your free 30-day trial without affecting your trial credits. Free plan services also continue running after the trial has expired.
+
+Trial features and limitations
+-------------------------------
+
+The 30-day trial gives you full access to the `Aiven Console <https://console.aiven.io/>`_, :doc:`API </docs/tools/api>`, :doc:`CLI </docs/tools/cli>`, :doc:`Aiven Terraform provider </docs/tools/terraform>`, and :doc:`Aiven Operator for Kubernetes </docs/tools/kubernetes>`. The credits can be used on any service types and plan sizes.
```
Comment (carabaestlein) [inline docs/platform/concepts/free-trial.rst:11]: You also have full access to the console etc without the 30 day trial. The key is you get credits to create non-free services, plus the ability to interact with functionality that requires non-free services (such as creating and placing services into VPCs).

**Review by carabaestlein — COMMENTED** [review]
Left some comments :)

**Comment by staceysalamon-aiven** [comment]
> Left some comments :)

Thank you for reviewing! I've incorporated your suggestions now.

**Review by carabaestlein — APPROVED** [review]
LGTM!
